### PR TITLE
micrometres for measured particle size now unit prefix is available

### PR DESCRIPTION
**Pull request by envirocoder**

### CODE REVIEW

File: 10314.xml [hunk 1]
```diff
@@ -68,7 +68,7 @@ TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR TH
 				<Type>String</Type>
 				<RangeEnumeration></RangeEnumeration>
 				<Units></Units>
-				<Description><![CDATA[Measurement Units Definition e.g. "Cel" for Temperature in degrees Celsius.]]></Description>
+				<Description><![CDATA[Measurement Units Definition e.g. "ug/m3" for concentration in micrograms per cubic metre.]]></Description>
```
Comment (jpradocueva) [inline 10314.xml:71]: @envirocoder, Reusable Resource (RR) 5701 has been already registered with OMA it is not possible to modify the description of an already registered RR. A new Reusable Resource will have to be created if you need a different description.

File: 10314.xml [hunk 2]
```diff
@@ -97,7 +97,7 @@ TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR TH
 				<Mandatory>Mandatory</Mandatory>
 				<Type>Float</Type>
 				<RangeEnumeration></RangeEnumeration>
-				<Units>m</Units>
+				<Units>um</Units>
```
Comment (jpradocueva) [inline 10314.xml:100]: @envirocoder  "um" is not a unit defined in SenML registry: https://www.iana.org/assignments/senml/senml.xhtml.
I suggest that you should join IPSO conference call to discuss this registration. Alternatively, please engage via this PR with the Working Group, they will provide further suggestions.
If you want me to send you the link for the meeting on Thursday please let me know, you can email to helpdesk@omaorg.org

File: 10314.xml (hunk 1 above)
Comment (jpradocueva) [reply 10314.xml:71]: The following Objects are affected by these changes:
* 10254
* 10258
* 10311
* 10314 (object in this registration)
* 3300
* 3301
* 3303
* 3304
* 3308
* 3311
* 3313 to 3331
* 3334
* 3335
* 3346

Comment (envirocoder) [reply 10314.xml:71]: It's fine, apologies, I wasn't thinking clearly when I changed it, I'd forgotten this wasn't my Resource! Doesn't really need changing.

File: 10314.xml (hunk 2 above)
Comment (envirocoder) [reply 10314.xml:100]: This change was based on a comment here:
https://github.com/OpenMobileAlliance/lwm2m-registry/pull/190#pullrequestreview-301021002
I've emailed requesting an invite. Thansk.

File: 10314.xml (hunk 1 above)
Comment (mkgillmore) [reply 10314.xml:71]: @envirocoder Please see the comments above.   Reusable Resources cannot be modified once used by accepted objects.   Also,  the group prefers the unit m for the measured particle size.

Thanks

Matt

**Pull request closed without merging**